src/main/java/org/myrobotlab/service/InMoov2Hand.java

```diff
@@ -489,7 +489,7 @@ public LeapData onLeapData(LeapData data) {
     return data;
   }
 
-  public void onMoveHand(HashMap<String, Double> map) {
+  public void onMove(Map<String, Double> map) {

```

Comment:
same comment as above regarding backwards compatibility ..  we should add this as a new method and mark the only one as deprecated.

Reply:
:+1: